packages/message-relayer/hardhat.config.ts

```diff
@@ -8,7 +8,7 @@ const config: HardhatUserConfig = {
     sources: './test/test-contracts',
   },
   solidity: {
-    version: '0.7.6',
+    version: '0.8.7',
```

Comment:
I think this use the same optimizer settings as the ones in the contracts package.

Reply:
To ensure consistency across packages, it might be good to export those settings as an object in `core-utils`, but that could be a separate PR.

Reply:
The `contracts` package is on 0.8.7 and the `message-relayer` here is on 0.8.7. Where is the problem?

Reply:
It specifies some [custom settings](https://github.com/ethereum-optimism/optimism/blob/7d0a411b2dccba402a07f2f6d84ba9eca21e89ce/packages/contracts/hardhat.config.ts#L60-L79), which I think we should strive to keep consistent on.